docs/grid_features/network_modifications.md

```diff
@@ -280,7 +280,7 @@ It is possible to specify a side of the element to connect. If no side is specif
 Class: `ConnectableConnection`
 
 ### Dangling line
-This modification is used to update the active and reactive powers of a dangling line.
+This modification is used to update the active and reactive powers of the load part of a dangling line.
```

Comment:
This documentation is for the network modification.
Can you also update the doc relative to the action (in `docs/simulation/security/index.md`)?